File: The Arcade VB/The Arcade VB/El ahorcado/IAhorcado.vb
Public Interface IAhorcado
    Function ElegirPalabra() As String
    Function GenerarEspacios(ByVal palabra As String) As Char()
    Sub ImprimirEspacios(ByVal longitud As Char())
    Function CompruebaUna(ByVal letra As String) As Boolean
    Sub ImprimirFinJuego(ByVal palabra As String, ByVal intentos As Integer, ByVal gano As Boolean)
End Interface

File: The Arcade/The Arcade/El ahorcado/IAhorcado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace The_Arcade.El_ahorcado
{
    interface IAhorcado
    {
        string ElegirPalabra();

        char[] GenerarEspacios(string palabra);

        void ImprimirEspacios(char[] longitud);

        bool CompruebaUna(string letra);

        void ImprimirFinJuego(string palabra, int intentos, bool gano);
    }
}
